server/svix-server/src/error.rs

```diff
@@ -422,3 +434,18 @@ impl IntoResponse for HttpError {
         (self.status, Json(self.body)).into_response()
     }
 }
+
+impl From<ErrorType> for Error {
+    fn from(typ: ErrorType) -> Self {
+        Self { trace: vec![], typ }
+    }
+}
+
+impl From<crate::core::webhook_http_client::Error> for Error {
```

Comment:
Not sure what this is for. More of a bookmark for myself than anything.

Reply:
b/c the webhook-http-client has its own Error type